src/components/TimePicker/styled/selectValue.js

```diff
@@ -32,111 +17,111 @@ const StyledSelectValue = styled.input.attrs(props => {
     line-height: normal;
 
     &::selection {
-        color: ${props => props.brandDarkContrastText};
-        background: ${props => props.brandDarkColor};
+        color: ${props => props.palette.getContrastText(props.palette.brand.dark)};
+        background: ${props => props.palette.brand.dark};
     }
 
     &::-moz-selection {
-        color: ${props => props.brandDarkContrastText};
-        background: ${props => props.brandDarkColor};
+        color: ${props => props.palette.getContrastText(props.palette.brand.dark)};
```

Comment:
here getContrastText is used twice with the same color, I think we should inject this value through styled.input.attrs in order to compute only once.